lingua_franca/lang/parse_en.py

```diff
@@ -77,7 +77,7 @@ def generate_plurals_en(originals):
 _STRING_LONG_ORDINAL_EN = invert_dict(_LONG_ORDINAL_EN)
 
 
-def _convert_words_to_numbers_en(text, short_scale=True, ordinals=False):
+def _convert_words_to_numbers_en(text, short_scale=True, ordinals=False, places=None):
```

Comment:
I can definitely understand shortening this to `places` in the function that is specifically handling decimal places, however here it's not 100% clear what places would refer to, so may be worth using the full `decimal_places` in these function definitions. 

Be good to update these docstrings while we're at it too.